newdle/api.py

```diff
@@ -278,6 +279,51 @@ def create_newdle(title, duration, timezone, timeslots, participants, private, n
     return NewdleSchema().jsonify(newdle)
 
 
+@api.route('/newdle/<code>', methods=('PATCH',))
+@use_args(EditNewdleSchema(partial=True), locations=('json',))
+def edit_newdle(args, code):
```

Comment:
We already have `update_newdle` for PATCH; let's reuse this and just add the missing code.